docs/explanation/e-hardening.md

```diff
@@ -0,0 +1,119 @@
+that the different users can be granted. 
+
+Juju user credentials must be stored securely and rotated regularly to prevent leakage and unauthorized access. 
+
+## Applications
+
+In the following we provide guidance on how to harden your deployment using:
+
+1. Operating System
+2. Kafka and ZooKeeper Security Upgrades
+3. Encryption 
+4. Authentication
+5. Monitoring and Auditing
+
+### Operating System
+
+Charmed Kafka and Charmed ZooKeeper currently run on top of Ubuntu 22.04. Deploy a [Landscape Client Charm]() in order to 
+connect the underlying VM to a Landscape User Account to manage security upgrades and integrate Ubuntu Pro subscriptions. 
+
+### Kafka and ZooKeeper Security Upgrades
+
+Charmed Kafka and Charmed ZooKeeper operators install a pinned revision of the [Charmed Kafka snap](https://snapcraft.io/charmed-kafka)
+and [Charmed ZooKeeper snap](https://snapcraft.io/charmed-zookeeper), respectively, in order to provide reproducible and secure environments. 
+New versions of Charmed Kafka and Charmed ZooKeeper may be released to provide patching of vulnerabilities (CVEs). 
+It is important to refresh the charm regularly to make sure the workload is as secure as possible. 
+For more information on how to refresh the charm, please refer to the [how-to upgrade](https://charmhub.io/kafka/docs/h-upgrade) documentation.
+
+### Encryption
+
+Charmed Kafka must be deployed with security enabled. This requires to deploy Kafka and ZooKeeper and relate them to one of the TLS certificate operator charms. 
+Please refer to the [Charming Security page](https://charmhub.io/topics/security-with-x-509-certificates) for more information on how to select the right certificate
+provider for your use-case. 
+
+For more information, please refer to the [how to enable encryption](https://charmhub.io/kafka/docs/h-enable-encryption) user guide.
+
+### Authentication
+
+Charmed Kafka supports the following authentication layers:
+
+1. [SCRAM-based SASL Authentication](/t/charmed-kafka-how-to-manage-app/10285)
+2. [certificate-base Authentication (mTLS)](/t/create-mtls-client-credentials/11079)
+3. OAuth Authentication using [Hydra](/t/how-to-connect-to-kafka-using-hydra-as-oidc-provider/14610) or [Google](/t/how-to-connect-to-kafka-using-google-as-oidc-provider/14611)
+
+Each combination of authentication scheme and encryption is associated to the dedicated listener and it maps to a well-defined port. 
+Please refer to the [listener reference documentation](/t/charmed-kafka-documentation-reference-listeners/13264) for more information. 
+
+### Monitoring and Auditing
+
+Charmed Kafka provides native integration with the [Canonical Observability Stack (COS)](https://charmhub.io/topics/canonical-observability-stack).
+To reduce the blast radius of infrastructure disruptions, the general recommendation is to deploy COS and the observed application into 
```

Comment:
```suggestion
To reduce the damage of infrastructure disruptions, the general recommendation is to deploy COS and the observed application into 
```

Reply:
Uhm, I actually took this phrasing from observability docs: https://charmhub.io/topics/canonical-observability-stack/reference/best-practices